### ISSUE TITLE
Addon fails to copy all tabs

### ISSUE THREAD

**Issue by vonHabsi** [piroor/multipletab#217]
## Short description
When you multiselect a whole tabtree, and apply the Copy Tabs to Clipboard menu not all the tabs are copied, 
## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install MTH.
 3. Enable permissions for Copying tabs to clipboard
 4. Open a lot of tabs, say more than 20 in a subtree
5. Click on the parent tab, making sure it is collapsed to ensure children are automatically selected even if they are hidden, and right click and choose Copy To Clipboard and the required format.

## Expected result
The whole list should be copied

## Actual result
Most of the time less than 15 are copied even if there are a lot more.

## Environment

 * Platform (OS):  Debian 9 64bits
 * Version of Firefox: Nightly 64, Beta 63
 * Version (or revision) of Multiple Tab Handler:  2.1.4

**Comment by piroor** [comment]
I couldn't reproduce this problem with large number direct children tabs, but I reproduced similar problem with mixed collapsed/expanded subtree, like:

 * 1 (collapsed)
   * 1-1 (expanded)
     * 1-1-1
     * 1-1-2
   * 1-2 (collapsed)
     * 1-2-1
     * 1-2-2

In this case, I ctrl-click the tab "1" then only "1-1", "1-2", "1-2-1", and "1-2-2". "1-1-1" and "1-1-2" are not selected because "1-1" is detected as an expanded tree. By recent commits I changed the behavior to select all descendants of the clicked tab "1" as expacted. Is this same to the problem you saw?

**Comment by vonHabsi** [comment]
I haven't been actually checking the collapsed state of all the tabs, and to checking which ones have actually been copied and which ones have been missed.

To simplify matters I collapse the parent tab of the whole tree and count the total number of child tabs.

I then copy it into a text editor and check the number of lines copied to see if matches the child tab count.

On coming to see your response, I tested a parent tree with 397 children, grandchildren, etc and they were all copied.

I have also tested it in other profiles and I would say the problem is fixed now.

Thanks a lot.
